features/liquidity/components/LiquidityBreakdown.tsx

```diff
@@ -73,13 +74,34 @@ const PoolHeader = ({ tokenA, tokenB, priceBreakdown }: PoolHeaderProps) => (
   </Inline>
 )
 
+const SwapFeeElem = ({
+  protocolFee,
```

Comment:
i think you could default these to 0 instead of making them optional - what do you think `protocolFee = 0`

Reply:
Might conflict with Ben's suggestions.

Reply:
It's a smart contract thing, they might be undefined due to older contract versions. And Ben wants to display a different message if they are undefined.